app/Http/Requests/ProductValidation.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ProductValidation extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+    public function rules(): array
+    {
+        return [
+            'name' => ['required', 'unique:products,name,'.$this->productId. 'max:255'],
+            'brand' => ['required', 'max:255'],
+            'color' => ['required', 'max:255'],
+            'size' => ['required', 'max:255'],
```

Comment:
check which key are you receiving in request.
accordingly the value use the laravel validation.